apps/src/templates/teacherDashboard/AssignmentVersionSelector.jsx

```diff
@@ -16,118 +17,137 @@ import {assignmentCourseVersionShape} from './shapes';
 const menuItemWidth = _(columnWidths).values().reduce(_.add);
 const menuWidth = menuItemWidth + 2 * STANDARD_PADDING;
 
-export default class AssignmentVersionSelector extends Component {
-  static propTypes = {
-    dropdownStyle: PropTypes.object,
-    onChangeVersion: PropTypes.func.isRequired,
-    selectedCourseVersionId: PropTypes.number,
-    courseVersions: PropTypes.objectOf(assignmentCourseVersionShape),
-    disabled: PropTypes.bool,
-    rightJustifiedPopupMenu: PropTypes.bool,
-  };
+export function AssignmentVersionSelector({
```

Comment:
Converted to a functional component since that makes it easier to wrap with the regional wrapper.

Also, we should leave things better than we found them sometimes.